examples/connector/golang/scripts/copy_protos.sh

```diff
@@ -1 +1 @@
-cp ../../../*.proto proto/
+cp ../../../*v2.proto proto/
```

Comment:
we should probably include the `_` too, like this: `cp ../../../*_v2.proto proto/`